napari/_qt/qt_viewer.py

```diff
@@ -9,6 +9,8 @@
 from qtpy.QtGui import QCursor, QGuiApplication
 from qtpy.QtWidgets import QFileDialog, QSplitter, QVBoxLayout, QWidget
 
+# I know this probably not preferred, left it as is temporarily.
+from ..components._viewer_key_bindings import *  # noqa: F403, F401
```

Comment:
Let's remove this and make the binding look at the shortcuts omn that file directly so that the decorator does not provide a binding, but we look at that from the settings.

Reply:
+1, that's a good idea.